lib/hint.rb

```diff
@@ -83,5 +83,10 @@ def share_html_contents(options)
       assets_dir = File.join(dir, "assets/")
       "Share the file: '#{index_file}' and the directory: '#{assets_dir}' as needed."
     end
+
+    def serve_public_option(_options)
+      "The --public option makes EVERY description in ~/.machinery publicly available! Be " \
+        "careful if you have descriptions, what should not be available for everyone."
```

Comment:
That reads a bit rough imo. Maybe `The --public option makes ALL of your system descriptions publicly available. Take care if there are system descriptions that should not be read by others.`?